src/main/kotlin/com/kotlindiscord/bot/extensions/HelpExtension.kt

```diff
@@ -27,13 +31,17 @@ class HelpExtension(kdBot: KDBot) : Extension(kdBot) {
             logger.debug { "Message length : ${messageArray.size}" }
 
             if (messageArray.isEmpty()) {
-                message.channel.createEmbed {
-                    title = "Command Help"
-                    description = formatMainHelp(gatherCommands())
-                }
+                Paginator(
+                    kdBot,
+                    message.channel,
+                    "Get Help",
+                    formatMainHelp(gatherCommands()),
+                    timeout = 10_000L,
+                    keepEmbed = true
+                ).send()
             } else {
                 message.channel.createEmbed {
-                    title = "Command Help"
+                    title = "Get Help"
```

Comment:
If it's not command help, then perhaps just help? I don't see why not command help or commands, though.